plan/entities.go

```diff
@@ -69,6 +98,22 @@ func (b Behaviors) Len() int           { return len(b) }
 func (b Behaviors) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
 func (b Behaviors) Less(i, j int) bool { return b[i].Name < b[j].Name }
 
+func (b Behaviors) validateAndApplyFilters(filtersByBehavior map[string][]Filter) error {
+	for i, behavior := range b {
+		filters := filtersByBehavior[behavior.Name]
+		for _, filter := range filters {
```

Comment:
Nit: `if len(filters) == 0 { continue }` to avoid changing the current behavior if it doesn't have filters